exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 50;
+let numberOfMentors = 20;
+let result = numberOfStudents + numberOfMentors;
```

Comment:
Use `const` when your variable value won't change at all.
Use `let` when the content of your variable will change later in the code.

Therefore, here you can change all the `let` to `const`.